package.json

```diff
@@ -47,13 +47,13 @@
     "lodash": "^4.17.21",
     "moment-range": "^4.0.2",
     "moment-timezone": "^0.5.37",
-    "mongoose": "^6.5.3",
+    "mongoose": "^6.11.3",
     "mongoose-id": "^0.1.3",
     "mongoose-paginate-v2": "^1.7.0",
     "pino": "^8.4.2",
-    "rss-parser": "^3.12.0",
+    "rss-parser": "^3.13.0",
```

Comment:
The new version of rss-parser (3.13.0) should be verified for any changes in its API or functionality that might affect existing code, considering it is a major update from 3.12.0.